codec-http/src/main/java/io/netty/handler/codec/http/multipart/InterfaceHttpPostRequestDecoder.java

```diff
@@ -20,7 +20,8 @@
 import java.util.List;
 
 /**
- * This decoder will decode Body and can handle POST BODY (or for PUT, PATCH or OPTIONS).
+ * This decoder will decode Body and can handle POST BODY (or for PUT, PATCH or OPTIONS or any allowed in RFC 7231
+ * except TRACE).
```

Comment:
Consider rewording this sentence. The way it is worded makes it sound like TRACE is allowed in RFC 7231 but we just don't support it.